### PR TITLE
Document bootstrapping and replica creation

### PR DESCRIPTION
**Pull request by alexeyklyukin**
#454

### CODE REVIEW

File: docs/replica_bootstrap.rst [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Patorni allows customizing creation of a new replica. It also supports defining what happens when the new empty cluster
+is being bootstrapped. The distinction between two is well defined: Patroni creates replicas only if the ``initialize``
+key is present in Etcd for the cluster. If there is no ``initialize`` key - Patroni calls bootstrap exclusively on the
+first node that takes the initialize key lock.
+
+Bootstrap
+=========
+
+PostgreSQL provides ```initdb`` command to initialize a new cluster and Patroni calls it by default. In certain cases,
```
Comment (cbandy) [inline docs/replica_bootstrap.rst]: Extra backtick here.

File: docs/replica_bootstrap.rst [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+one of the ``recovery_target_*`` parameters, together with the ``recovery_target_timeline`` set to ``promote``.
+
+ .. note:: Bootstrap methods are neither chained, nor fallen-back to the default one
+
+
+
+Building replicas
+=================
+
+Patroni uses tried and proven``pg_basebackup`` in order to create new replicas. One downside of it is that it requires
```
Comment (cbandy) [inline docs/replica_bootstrap.rst]: Need a space after "proven" to render correctly.

**Comment by alexeyklyukin** [comment]
You can view the compiled version of the documentation at http://patroni.readthedocs.io/en/docs-bootstrap/, particularly these changes: http://patroni.readthedocs.io/en/docs-bootstrap/replica_bootstrap.html

File: docs/SETTINGS.rst [hunk 3]
```diff
@@ -23,6 +23,9 @@ Bootstrap configuration
         -  **use\_slots**: whether or not to use replication_slots. Must be False for PostgreSQL 9.3. You should comment out max_replication_slots before it becomes ineligible for leader status.
         -  **recovery\_conf**: additional configuration settings written to recovery.conf when configuring follower. 
         -  **parameters**: list of configuration settings for Postgres. Many of these are required for replication to work.
+-  **method**: custom script to use for bootstrpapping this cluster.
+   See :ref:`custom bootstrap methods documentation <custom_bootstrap>` for details.
+   When ``initdb`` is specified revert to the default ``initdb`` command.
```
Comment (CyberDem0n) [inline docs/SETTINGS.rst]: If the method is not specified `initdb` will also be used.

Comment (alexeyklyukin) [inline docs/SETTINGS.rst]: That's why I write it is the default. I will make it more clear.

**Comment by alexeyklyukin** [comment]
👍

**Comment by CyberDem0n** [comment]
:+1:

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**